### PR TITLE
Add 5.1.0.rc1 release post

### PR DESCRIPTION
**Pull request by rafaelfranca**

### CODE REVIEW

File: _posts/2017-03-20-Rails-5-1-rc1.markdown [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+gem install rails -v 5.1.0.rc1
+```
+
+Make sure you are using RubyGems 2.6.11 or later before trying to install a pre release, with:
+
+```
+gem update --system 2.6.11
+```
+
+Thank you for [Samuel Giddins](https://github.com/segiddins) for releasing this new version of RubyGems just in time to
```
Comment (claudiob) [inline _posts/2017-03-20-Rails-5-1-rc1.markdown]: @rafaelfranca Thank you _to_ Samuel Giddins

Comment (rafaelfranca) [reply _posts/2017-03-20-Rails-5-1-rc1.markdown]: Fixed

File: _posts/2017-03-20-Rails-5-1-rc1.markdown [hunk 2]
```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: 'Rails 5.1.0.rc1: Loving JavaScript, System Tests, Encrypted Secrets, and more'
+category: releases
+author: rafaelfranca
+published: true
+date: 2017-03-20 14:00 -04:00
+---
+
+We're happy to announce Rails 5.1.0.rc1 has been released. With the help of the community we to polished the Rails 5.1
```
Comment (kaspth) [inline _posts/2017-03-20-Rails-5-1-rc1.markdown]: Strike `to` in `we to polished` 😊

File: _posts/2017-03-20-Rails-5-1-rc1.markdown [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+release with more than [380 commits](https://github.com/rails/rails/compare/v5.1.0.beta1...5.1.0.rc1).
+
+While we are working to close the last issues, you can check the [the Rails 5.1 releases notes](http://edgeguides.rubyonrails.org/5_1_release_notes.html),
```
Comment (kaspth) [inline _posts/2017-03-20-Rails-5-1-rc1.markdown]: Extra `the` in `you can check the [the`

File: _posts/2017-03-20-Rails-5-1-rc1.markdown [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+* [Action Mailer CHANGELOG](https://github.com/rails/rails/blob/v5.1.0.rc1/actionmailer/CHANGELOG.md)
+* [Action Pack CHANGELOG](https://github.com/rails/rails/blob/v5.1.0.rc1/actionpack/CHANGELOG.md)
+* [Action View CHANGELOG](https://github.com/rails/rails/blob/v5.1.0.rc1/actionview/CHANGELOG.md)
+* [Active Model CHANGELOG](https://github.com/rails/rails/blob/v5.1.0.rc1/activemodel/CHANGELOG.md)
+* [Active Record CHANGELOG](https://github.com/rails/rails/blob/v5.1.0.rc1/activerecord/CHANGELOG.md)
+* [Active Support CHANGELOG](https://github.com/rails/rails/blob/v5.1.0.rc1/activesupport/CHANGELOG.md)
+* [Active Job CHANGELOG](https://github.com/rails/rails/blob/v5.1.0.rc1/activejob/CHANGELOG.md)
+* [Railties CHANGELOG](https://github.com/rails/rails/blob/v5.1.0.rc1/railties/CHANGELOG.md)
+
+[Basecamp 3](https://basecamp.com) is already running this release candidate in production. Our friends of
```
Comment (kaspth) [inline _posts/2017-03-20-Rails-5-1-rc1.markdown]: `Our friends *at*` 😊

File: _posts/2017-03-20-Rails-5-1-rc1.markdown [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+gem install rails -v 5.1.0.rc1
+```
+
+Make sure you are using RubyGems 2.6.11 or later before trying to install a pre release, with:
+
+```
+gem update --system 2.6.11
+```
+
+Thank you to [Samuel Giddins](https://github.com/segiddins) for releasing this new version of RubyGems just in time to
```
Comment (kaspth) [inline _posts/2017-03-20-Rails-5-1-rc1.markdown]: Replace `to` with `for` in `in time to`.

**Review by kaspth — COMMENTED** [review]
❤️🤤

File: _posts/2017-03-20-Rails-5-1-rc1.markdown (hunk 2 above)
Comment (rafaelfranca) [reply _posts/2017-03-20-Rails-5-1-rc1.markdown]: Fixed

File: _posts/2017-03-20-Rails-5-1-rc1.markdown [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+layout: post
+title: 'Rails 5.1.0.rc1: Loving JavaScript, System Tests, Encrypted Secrets, and more'
+category: releases
+author: rafaelfranca
+published: true
+date: 2017-03-20 15:00 -04:00
+---
+
+We're happy to announce Rails 5.1.0.rc1 has been released. With the help of the community we polished the Rails 5.1
+release with more than [380 commits](https://github.com/rails/rails/compare/v5.1.0.beta1...5.1.0.rc1).
```
Comment (claudiob) [inline _posts/2017-03-20-Rails-5-1-rc1.markdown:11]: @rafaelfranca this URL is missing a `v` as reported by @javan -- should be https://github.com/rails/rails/compare/v5.1.0.beta1...v5.1.0.rc1 instead

Comment (claudiob) [reply _posts/2017-03-20-Rails-5-1-rc1.markdown:11]: meaning `https://github.com/rails/rails/compare/v5.1.0.beta1...v5.1.0.rc1`

**Pull request merged**